### PR TITLE
Expose hasher to apps, and remove hashedAsset helper use

### PR DESCRIPTION
**Pull request by adgad**
@wheresrhys This look alright?

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -98,7 +98,9 @@ module.exports = options => {
 		next();
 	});
 
-	let hasher = str => str;
+	let hasher = {
```
Comment (adgad) [inline node/index.js]: wasn't sure about this and where it's used, but I assume it should have the same API as the actual hasher...

**Comment by adgad** [comment]
And consumed here - https://github.com/Financial-Times/next-article/pull/1791

File: node/index.js (hunk 1 above)
Comment (wheresrhys) [inline node/index.js]: Would you mind deleting https://github.com/Financial-Times/n-ui/blob/0caf851bce8265fbf12b00160c7d8b050398d944/node/index.js#L115.

Also I think it should be exposed on the app as just a function with a more descriptive name... `getHashedAssetUrl`?
(most of the code in the assets bits and pieces I imagine will be refactored massively as it's a mess, so it shouldn't really inform what the public api looks like)

**Comment by wheresrhys** [comment]
Just realised this would completely unstyle ie8!!!

This conditional comment needs tweaking https://github.com/Financial-Times/n-ui/blob/26536bb4da0834c5d757b68f1935e549c92223f0/layout/partials/stylesheets.html#L1

**Comment by adgad** [comment]
@wheresrhys I *think* this does the trick. Some of the styles look a bit naff on IE8 (cropping of the FT logo etc), I suspect we had those covered in main-ie8.css which no longer exists.

**Comment by wheresrhys** [comment]
yeah -IE8 is technically no longer core experience, so we have the go ahead to be a bit broken

**Pull request merged**